pages/itemdetails/_itemId.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div>
+    <v-img :src="imageLink(Item.Id, 'backdrop')"></v-img>
+    <h1>{{ Item.Name }}</h1>
+    <p>{{ Item.Overview }}</p>
+    <v-btn color="primary">Play {{ Item.Type }}</v-btn>
+    <v-btn>More</v-btn>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+export default Vue.extend({
+  data() {
+    return {
+      Name: '',
+      Item: {},
+      Seasons: {},
+      Episodes: []
```

Comment:
```suggestion
      name: '',
      item: {},
      seasons: {},
      episodes: []
```

I don't think anything uses capitalized properties. For things coming from the API, we don't really have a choice at the moment for the children, but I expect them to be going for camelCase with the rewrite, so it makes sense to keep our parent properties as camelCase as well.

Reply:
Ok, that makes sense, I'll update it.